static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -38,29 +38,34 @@ const tags: Record<string, unknown> = {
 	bundle: 'standalone',
 };
 
-const commercialModules: Modules = [
+// modules necessary to load an ad
+const commercialBaseModules: Modules = [
 	['cm-setAdTestCookie', setAdTestCookie],
+	['cm-prepare-prebid', preparePrebid],
+	// Permutive init code must run before google tag enableServices()
+	// The permutive lib however is loaded async with the third party tags
+	['cm-prepare-googletag', () => initPermutive().then(prepareGoogletag)],
+	['cm-prepare-a9', prepareA9],
+	['cm-mobileSticky', initMobileSticky],
+	['cm-highMerch', initHighMerch],
+	['cm-articleAsideAdverts', initArticleAsideAdverts],
+	['cm-articleBodyAdverts', initArticleBodyAdverts],
+	['cm-liveblogAdverts', initLiveblogAdverts],
```

Comment:
As discussed: these should only be loaded if the user is **not** ad-free. :)

Maybe we can omit the `setAdTestCookie` for ad-free users, too.